app/models/cinstance.rb

```diff
@@ -172,9 +172,7 @@ def self.by_service(service)
     if service == :all || service.blank?
       all
     else
-      # NOTE: `reorder(nil)` resets the order set in the Plan's default scope,
-      # because ORDER BY in subqueries fails in Oracle with 'ORA-00907: missing right parenthesis'.
-      where { plan_id.in( my { Plan.issued_by(service).select(:id).reorder(nil)} ) }
+      where { plan_id.in( my { Plan.issued_by(service).select(:id)} ) }
```

Comment:
I wouldn't remove the `reorder(nil)` from these queries. IMO keep it. It is solid to stay. But also fine to remove if you really want.

Reply:
After removing the default scope I don't think it's needed. And if someone ever wants to add the default scope, or add some order in subqueries, the test should fail.

Reply:
I reverted the `reorder(nil)`, but the spaces changes have left :grimacing: